verify/boilerplate/boilerplate.py

```diff
@@ -86,7 +86,7 @@ def file_passes(filename, refs, regexs):
         (data, found) = p.subn("", data, 1)
 
     # remove shebang from the top of shell files
-    if extension == "sh":
+    if extension == "sh" or extension == "py":
```

Comment:
I have a script that uses unbuffered (-u), which there doesn't seem to be a good way to programmatically use without setting the command line arg.